code/game/objects/items/explosives/plastique.dm

```diff
@@ -18,7 +18,7 @@
 	/// smoke type created when the c4 detonates
 	var/datum/effect_system/smoke_spread/smoketype = /datum/effect_system/smoke_spread/bad
 	/// radius this smoke will encompass
-	var/smokeradius = 2
+	var/smokeradius = 1
 
```

Comment:
???

Reply:
>C4 now creates less smoke, for a much shorter duration. It was kinda shitty for breaching because you'd have to wait for days for the smoke to clear, or charge through blind.

Basically if you want to actually use it to breach a building with hostiles in it, you have to wait 2 years for a cloud of smoke to clear, which makes C4 really bad.

Reply:
removed.